app/build.gradle

```diff
@@ -550,7 +550,7 @@ dependencies {
     androidTestImplementation Deps.androidx_junit
     androidTestImplementation Deps.androidx_work_testing
     androidTestImplementation Deps.mockwebserver
-
+    implementation("com.squareup.okhttp3:okhttp:4.9.0")
```

Comment:
Drive-by: Is this really needed? 🤔 - Since we are not using (and do not want to use) this in the app, I wonder if `testImplementation` would be enough. But still curious if needed at all?

Reply:
I needed that while I was debugging to get the tests running, but you are right, looks like we can remove this...